Tests.Integration.Presentation.Web/SystemUsage/V2/ItSystemUsageApiV2Test.cs

```diff
@@ -827,6 +826,60 @@ await ItSystemUsageV2Helper.SendPatchGDPR(token, usageDTO.Uuid, gdprVersion3)
             gdprResponse = dto.GDPR;
             AssertGDPR(gdprVersion3, gdprResponse);
         }
+        
+
+        [Fact]
+        public async Task Can_PATCH_Validity()
+        {
+            //Arrange
+            var (token, user, organization, system) = await CreatePrerequisitesAsync();
+
+            var date1 = DateTime.UtcNow;
+            var lifeCycle1 = LifeCycleStatusChoice.Operational;
+            var date2 = DateTime.UtcNow.AddDays(-1 - A<int>());
+            var lifeCycle2 = LifeCycleStatusChoice.Undecided;
+
+            var generalVersion1 = CreateValidityInput(date1, lifeCycle1);
+            var generalVersion2 = CreateValidityInput(date2, lifeCycle2);
+            var generalVersion3 = new GeneralDataUpdateRequestDTO();
+            
+            var usageDTO = await ItSystemUsageV2Helper.PostAsync(token, CreatePostRequest(organization.Uuid, system.Uuid));
+
+            //Act
+            await ItSystemUsageV2Helper.SendPatchGeneral(token, usageDTO.Uuid, generalVersion1)
+                .WithExpectedResponseCode(HttpStatusCode.OK)
+                .DisposeAsync();
+
+            //Assert version 1
+            var dto = await ItSystemUsageV2Helper.GetSingleAsync(token, usageDTO.Uuid);
+            var generalResponse = dto.General.Validity;
+            var expectedDateValidity = true;
+            var expectedLifeCycleValidity = true;
+            AssertValidity(generalVersion1.Validity, generalResponse, expectedDateValidity, expectedLifeCycleValidity);
+
+            //Act
+            await ItSystemUsageV2Helper.SendPatchGeneral(token, usageDTO.Uuid, generalVersion2)
+                .WithExpectedResponseCode(HttpStatusCode.OK)
+                .DisposeAsync();
+
+            //Assert version 2
+            dto = await ItSystemUsageV2Helper.GetSingleAsync(token, usageDTO.Uuid);
+            generalResponse = dto.General.Validity;
+            expectedDateValidity = false;
+            expectedLifeCycleValidity = false;
+            AssertValidity(generalVersion2.Validity, generalResponse, expectedDateValidity, expectedLifeCycleValidity);
+
+            //Act - reset
+            await ItSystemUsageV2Helper.SendPatchGeneral(token, usageDTO.Uuid, generalVersion3)
+                .WithExpectedResponseCode(HttpStatusCode.OK)
+                .DisposeAsync();
+
+            //Assert version 3 - properties should have been reset
+            dto = await ItSystemUsageV2Helper.GetSingleAsync(token, usageDTO.Uuid);
+            generalResponse = dto.General.Validity;
+            expectedDateValidity = true;
```

Comment:
since you use variables for your test, remember to also set expectedLifeCycleValidity so that you dont have to look in the other block to find the expected value... you could also just use name parameters in your method call is readability is your concern. This certaintly takes up a lot of space in the file